### PR TITLE
*flip

### PR DESCRIPTION
**Pull request by Amelia0010**
Can't flip while stunned, and flips might fail, stunning you.

#### Changelog

:cl:  
rscadd: *flip
/:cl:

### CODE REVIEW

**Comment by Amelia0010** [comment]
Oh and it's shamelessly taken from here: https://github.com/ParadiseSS13/Paradise/pull/8001

File: code/modules/mob/living/emote.dm [hunk 1]
```diff
@@ -111,9 +111,15 @@
...
-				src.SpinAnimation(7,1)
-				m_type = 2
+			if(!restrained() || !resting || !sleeping || !weakened)
+				if(prob(5))
+					message = "<B>[src]</B> attempts a flip and crashes to the floor!"
+					SpinAnimation(5,1)
+					sleep(3)
+					Weaken(2)
+				else
+					src.SpinAnimation(7,1)
```
Comment (ShadowDeath6) [inline code/modules/mob/living/emote.dm:121]: what do these numbers mean

Comment (Amelia0010) [reply code/modules/mob/living/emote.dm:121]: 7/5 are the amount of ticks in the spin, 1 is the amount of spins. It sleeps for 3 seconds to get you through some of the spin, then stuns you when it looks like you're horizontal, so you're not standing vertically then suddenly on the ground.

**Comment by 1fbff5f83b23d39d38b1dfcb4cac8d9b** [comment]
Fun removal

**Comment by Koffheatalt** [comment]
fun police
make it only fail if you are fat

**Comment by Amelia0010** [comment]
This isn't a fun REMOVAL, it's a fun ADDITION. 

Now you get to laugh at murderboners when they get killed by an assistant because they *flip'd in combat and got stunned

**Comment by Notamaniac** [comment]
does this make the golden bikehorn op now?

**Comment by Alexandria404** [comment]
Great PR. Like it.

**Comment by Zostroll** [comment]
Does this effect duel esword flipping?

**Comment by Amelia0010** [comment]
no. when you dual e-sword flip, you aren't typing *flip in your chat, are you?

**Comment by ShadowDeath6** [comment]
alt+clicking a double esword makes it so you do the flip emote sometimes when you attack

**Comment by Zostroll** [comment]
I see no reason for this PR to exist other than to as the tag states REMOVE FUN.

**Comment by grubstank** [comment]
I like it. It makes being on the receiving end of flips even more degrading.

**Pull request closed without merging**